### DISCUSSION TITLE
Convention for NumPy: `onp` vs `np` in Flax

### DISCUSSION THREAD

**Discussion by 8bitmp3** [google/flax#730] [Ideas] [answered]
@avital I noticed the convention for ordinary NumPy is `onp` in some places in `google/flax` (https://github.com/google/flax/search?q=onp). Should we migrate to `np`, which is similar to this PR: https://github.com/google/jax/pull/3760 by @jakevdp? HuggingFace appears to be also using `np` in its JAX Transformers with Flax.

```py
import numpy as np
import jax.numpy as jnp
```

Cheers 👍

**Comment by avital** [comment, marked as answer]
Nice catch -- indeed we should switch away from using `onp`. Feel free to file an issue or pull request. Thanks!
